server/src/main/java/org/elasticsearch/action/search/OpenPointInTimeRequest.java

```diff
@@ -20,16 +21,18 @@
 import org.elasticsearch.tasks.TaskId;
 
 import java.io.IOException;
+import java.util.Arrays;
 import java.util.Map;
 import java.util.Objects;
 
 import static org.elasticsearch.action.ValidateActions.addValidationError;
 
 public final class OpenPointInTimeRequest extends ActionRequest implements IndicesRequest.Replaceable {
+
     private String[] indices;
     private IndicesOptions indicesOptions = DEFAULT_INDICES_OPTIONS;
     private TimeValue keepAlive;
-
+    private int maxConcurrentShardRequests = 5;
```

Comment:
nit: shall we link to the default constant for the same parameter in search, given the two have the same default value for now?

Reply:
Yes, I pushed https://github.com/elastic/elasticsearch/pull/96782/commits/a252899ac24b62c7624aa5051f7fea272b5f8ea5.